lib/src/contextual-menu/ItemAction.tsx

```diff
@@ -82,10 +83,12 @@ const Label = styled.span`
 `;
 
 const Text = styled.span`
-  color: ${CoreTokens.color_grey_900};
-  font-family: ${CoreTokens.type_sans};
-  font-size: ${CoreTokens.type_scale_02};
-  line-height: 24px;
+  color: ${({ theme }) => theme.menuItemFontColor};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme }) => theme.menuItemFontSize};
+  font-style: ${({ theme }) => theme.menuItemFontStyle};
+  font-weight: ${({ theme }) => theme.menuItemFontWeight};
```

Comment:
`font-weight` is overwritting the value given in `Action`

Reply:
Yes, my fault, removed!